File: app/src/main/java/forpdateam/ru/forpda/api/qms/interfaces/IQmsChatItem.java
package forpdateam.ru.forpda.api.qms.interfaces;

/**
 * Created by radiationx on 03.08.16.
 */
public interface IQmsChatItem {
    boolean isMyMessage();

    int getId();

    boolean getReadStatus();

    String getTime();

    String getAvatar();

    String getContent();

    boolean isDate();

    String getDate();
}

File: app/src/main/java/forpdateam/ru/forpda/entity/remote/qms/IQmsChatItem.kt
package forpdateam.ru.forpda.entity.remote.qms

/**
 * Created by radiationx on 03.08.16.
 */
interface IQmsChatItem {
    val isMyMessage: Boolean
    val id: Int
    val readStatus: Boolean
    val time: String?
    val avatar: String?
    val content: String?
    val isDate: Boolean
    val date: String?
}
